src/Config.js

```diff
@@ -190,6 +192,16 @@ export class Config {
       ) {
         throw 'passwordPolicy.maxPasswordHistory must be an integer ranging 0 - 20';
       }
+
+      if (
+        passwordPolicy.resetTokenReuseIfValid &&
+        typeof passwordPolicy.resetTokenReuseIfValid !== 'boolean'
+      ) {
+        throw 'resetTokenReuseIfValid must be a boolean value';
```

Comment:
Can you please add test cases for the validation as well?

Reply:
I sure can, will do in the AM.

Reply:
Can you also please take a look at the coverage, there seem to be some lines that are not captures by tests, such as  `expiresDate = new Date(expiresDate.iso);`?

Reply:
Thanks, but I had to add that line specifically to get the tests to pass on postgres, so I'm surprised that it's not captured. Is that at all related to a large amount of tests failing on postgres? And if so, do you think it's postgres being flaky, or something wrong with the latest commit that broke postgres? I'm not having any postgres issues locally, and can't seem to see any error traces on travis-ci.

Reply:
> I had to add that line specifically to get the tests to pass on postgres

Could you add a test case in which you feed in a date in the format that would be received from Postgres, so we can test the correct date conversion?

Reply:
It was related to the flaky tests, that line runs when `PARSE_SERVER_TEST_DB=postgres`, so it would be covered by the tests. Without the `expiresDate = new Date(expiresDate.iso);` line, the tests fail on postgres with:

```
1) Email Verification Token Expiration:  should match codes with emailVerifyTokenReuseIfValid
  Message:
    Expected 'U46kUHXB4WdNYz4GJhjxGIx8m' to equal 'u4T37e1FOpjiDkiDuVgkYD9OF'.
  Stack:
    Error: Expected 'U46kUHXB4WdNYz4GJhjxGIx8m' to equal 'u4T37e1FOpjiDkiDuVgkYD9OF'.
        at <Jasmine>
        at reconfigureServer.then.then.then.then.then.userAfterRequest (/home/travis/build/parse-community/parse-server/spec/EmailVerificationToken.spec.js:676:55)
        at process._tickCallback (internal/process/next_tick.js:68:7)
  Message:
    Expected $.iso = '2020-11-18T22:26:24.649Z' to equal '2020-11-18T22:26:24.667Z'.
  Stack:
    Error: Expected $.iso = '2020-11-18T22:26:24.649Z' to equal '2020-11-18T22:26:24.667Z'.
        at <Jasmine>
        at reconfigureServer.then.then.then.then.then.userAfterRequest (/home/travis/build/parse-community/parse-server/spec/EmailVerificationToken.spec.js:677:66)
        at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
It seems that it was a false negative, coverage looks ok now.